docs/learn/learn-xcm-pallet.md

```diff
@@ -0,0 +1,21 @@
+---
+id: learn-xcm-pallet
+title: XCM FRAME Pallet Overview
+sidebar_label: XCM FRAME Pallet Overview
+description: Learn how the XCM pallet works, and its place in the ecosystem.
+keywords: [cross-consensus, XCM, XCMP, interoperability, communication]
+slug: ../learn-xcm-pallet
+---
+
+The XCM pallet ([`pallet-xcm`](https://github.com/paritytech/polkadot/tree/master/xcm/pallet-xcm))
+provides a developer-friendly interface for more of the common XCM messages.
+
+This pallet provides default implementations for many traits required by `XcmConfig`, which is
+highly useful for configuration.
+
+It defines a set of ten extrinsics that can be utilized to build XCVM programs, either to target the
```

Comment:
Keep the wiki writeup generic. We won't be able to spot this if they add a 11th extrinsic :)

```suggestion
It defines a set of extrinsics that can be utilized to build XCVM programs, either to target the
```